### PR TITLE
Feature Progress bar

### PR DESCRIPTION
**Pull request by polomarcus**
Add remaining percentage of the coutdown.
Works with :  
* coutdown
* end-time

screen shot 2015-03-26 at 13 49 01

### CODE REVIEW

**Comment by polomarcus** [comment]
The gh-pages might does like this PR because of 
```HTML
    <script src="app/js/_timer.js"></script>
    <script src="app/js/i18nService.js"></script>
    <script src="app/js/progressBarService.js"></script>
````

I will make a PR to use https://github.com/tschaub/grunt-gh-pages and facilitate all of this

File: index.html [hunk 1]
```diff
@@ -14,7 +14,10 @@
     <script src="bower_components/momentjs/min/moment.min.js"></script>
     <script src="bower_components/momentjs/min/locales.min.js"></script>
     <script src="bower_components/humanize-duration/humanize-duration.js"></script>
-    <script src="dist/angular-timer.js"></script>
+    <!--<script src="dist/angular-timer.js"></script>-->
```
Comment (AaronBuxbaum) [inline index.html]: This line can be deleted

Comment (polomarcus) [inline index.html]: @siddii I think I am going to create a angular-timer-all.js for index.html that could be used in the gh-pages and for https://github.com/siddii/angular-timer/issues/168

Comment (siddii) [inline index.html]: Sure... Sounds good to me :)

**Comment by polomarcus** [comment]
With this new grunt work flow we have : 
* dist/index.html with compiled JS for gh-pages
* /index.html with non compiled JS
* grunt task gh-pages that will push the content of dist/ to gh-pages
* a file `dist/assets/js/angular-timer-all.min.js`is available for https://github.com/siddii/angular-timer/issues/168
* The test grunt task tests now the dist files

**Beware**  with /index.html the examples page will not work, as they are not essential to the dev phase. The examples works for dist/index.html

**Comment by AaronBuxbaum** [comment]
@siddii I think you need to also update your demo page, since this change breaks it. Looks good though, going to try it out tonight!

**Comment by polomarcus** [comment]
Sorry, i forget to include the gh-pages grunt task in my PR: https://github.com/siddii/angular-timer/pull/172

**Comment by polomarcus** [comment]
http://siddii.github.io/angular-timer/ updated with gh-pages grunt task

**Comment by siddii** [comment]
Awesome.. :+1:  Thanks @polomarcus, you rock!

**Comment by polomarcus** [comment]
No, no, no, you rock! haha

**Comment by AaronBuxbaum** [comment]
Works perfectly for my purposes (though I think the demo page is missing a binding to the 'active' progress bar); thanks a lot!

**Pull request merged**